pkg/controllers/membercluster/v1beta1/membercluster_controller.go

```diff
@@ -452,6 +452,14 @@ func (r *Reconciler) syncInternalMemberClusterStatus(imc *clusterv1beta1.Interna
 	r.aggregateJoinedCondition(mc)
 	// Copy resource usages.
 	mc.Status.ResourceUsage = imc.Status.ResourceUsage
+	// Copy additional conditions.
+	for idx := range imc.Status.Conditions {
```

Comment:
i remember we'll calculate and build the conditions by checking the imc conditions.  We cannot copy all? just copy what we need?